routing/router.go

```diff
@@ -1,43 +1,72 @@
 package routing
 
 import (
+
 	"net/http"
 
 	"github.com/gorilla/mux"
 
 	"github.com/kiali/kiali/config"
+
+	"github.com/kiali/kiali/log"
 )
 
 // NewRouter creates the router with all API routes and the static files handler
 func NewRouter() *mux.Router {
 
-	router := mux.NewRouter().StrictSlash(true)
+	conf := config.Get()
+	webRoot := conf.Server.WebRoot
+	webRootWithSlash := webRoot + "/"
+
+	rootRouter := mux.NewRouter().StrictSlash(false)
+	appRouter := rootRouter;
+
+	if webRoot == "/" {
+		// appRouter.StrictSlash(true)
```

Comment:
Do we need this empty if clause? It is probably better to just do the work (i.e. the work that is currently performed in the else clause) in an `if webRoot != "/"` clause instead:

```
if webRoot != "/" {
   ... what is in the else clause now would go here...
}
```

Reply:
that's an oversight. I'll resubmit